heat/core/tests/test_statistics.py

```diff
@@ -619,57 +619,51 @@ def test_maximum(self):
         self.assertTrue((maximum._DNDarray__array == torch.max(comparison1, comparison2)).all())
 
         # check maximum over float elements of split 3d tensors
-        # TODO: add check for uneven distribution of dimensions (see Issue #273)
-        size = ht.MPI_WORLD.size
         torch.manual_seed(1)
```

Comment:
Same question as in statistics - Can this be replaced with `ht.random.seed(1)` ?

Reply:
yes, see above